package.json

```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@duckduckgo/autoconsent": "^4.1.1",
-    "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#6.5.1",
+    "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#ema/mobile-password-generation-updates",
```

Comment:
this will be updated back to a stable release when it's available (before merging this PR)

Reply:
Updated now